src/migration/1609958100806-signature-entitity-table.ts

```diff
@@ -0,0 +1,25 @@
+import {MigrationInterface, QueryRunner} from 'typeorm';
+
+export class signatureEntitityTable1609958100806 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<any> {
+    await queryRunner.query(`
+            CREATE TABLE IF NOT EXISTS signatures_entities (
+                signature UUID,
+                entity UUID,
+                PRIMARY KEY(signature, entity),
+                CONSTRAINT fk_signature
+                    FOREIGN KEY(signature) 
+                        REFERENCES signatures(uuid),
+                CONSTRAINT fk_entity
+                    FOREIGN KEY(entity) 
+                        REFERENCES entities(uuid)
+            );
+        `);
```

Comment:
We should add indexes to these relationships as well.

Might it make sense to create this table using a typeorm entity itself?